sci-biology/salmon/salmon-0.5.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
I can't `repoman full` for the ebuilds I set to `EAPI=6` until it makes its way to prefix.

If you prefer tell me when everything else is right and then I'll just amend the commit before you merge.